FirebaseAI/Tests/TestApp/Tests/Integration/LiveSessionTests.swift

```diff
@@ -327,10 +249,7 @@ struct LiveSessionTests {
     await session.close()
   }
 
-  @Test(
-    .disabled("Temporarily disabled"),
```

Comment:
This test works locally though it sometimes flakes. I'll see if it passes on CI with the new models, otherwise we should disable it again or consider using [`withKnownIssue(isIntermittent: true)`](https://developer.apple.com/documentation/testing/known-issues#Handle-a-nondeterministic-failure).